changelogs/unreleased/fix-typing-of-decorators.yml

```
@@ -0,0 +1,6 @@
+---
+description: Fix the typing of agent cache and plugins decorators.
+change-type: patch
+sections:
+  bugfix: "{{description}}"
+destination-branches: [master, iso7]
```

```suggestion
destination-branches: [iso7]
```
Due to a protection mechanism of the merge tool, you have to exclude it from already-merged branches.